easybuild/framework/easyconfig/default.py

```diff
@@ -154,6 +154,7 @@
     'multi_deps': [{}, "Dict of lists of dependency versions over which to iterate", DEPENDENCIES],
     'multi_deps_load_default': [True, "Load module for first version listed in multi_deps by default", DEPENDENCIES],
     'osdependencies': [[], "OS dependencies that should be present on the system", DEPENDENCIES],
+    'moddependpaths': [None, "Absolute path or paths that should be searched for dependencies", DEPENDENCIES],
```

Comment:
I think we need to be more explicit about what this is, this could potentially be confusing (are we talking about EB or the module tool when we talk about dependencies?). How about something like:
```
Absolute path or paths to prepend to MODULEPATH before loading the resulting module file
```

Reply:
Hmmm.  Since it's included in the DEPENDENCIES section, shouldn't it be clear to an EC author what "dependencies" refers to?  How about

    Absolute path or paths to prepend to MODULEPATH before loading the module's dependencies